athlinks-schema.ttl

```diff
@@ -6,13 +6,24 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix schema: <http://schema.org/> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://adp.athlinks.com/2017/01/rdf-schema> a owl:Ontology ;
   owl:imports <http://topbraid.org/schema/> ;
   rdfs:label "The Athlinks Schema vocabulary".
 
 ### CLASSES
+athlinks:TerrainSpecifics a rdfs:Class;
+  rdfs:comment "Specifics about course terrain such as altitude or grade." ;
+  rdfs:label "Terrain Specifics" ;
+  rdfs:subClassOf schema:Landform ;
```

Comment:
I don't think we need a subclass of Landform here. 
Two options :
1. The additional properties can just have their domain set to Landform.
2. Landform has a property that it may be better to use for these additional properties called. http://schema.org/additionalProperty